spec/Plugin/QueryDefaultsPluginSpec.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace spec\Http\Client\Common\Plugin;
+
+use PhpSpec\Exception\Example\SkippingException;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\StreamInterface;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Psr\Http\Message\UriInterface;
+
+class QueryDefaultsPluginSpec extends ObjectBehavior
+{
+    public function it_is_initializable()
+    {
+        $this->beConstructedWith([]);
```

Comment:
you can add this to a `let` method, it's like setUp in phpunit. If you use beConstructedWith again, it will override the on in the let method.